src/pal/src/safecrt/internal_securecrt.h

```diff
@@ -23,6 +23,13 @@
 #ifndef _INC_INTERNAL_SECURECRT
 #define _INC_INTERNAL_SECURECRT
 
+#ifndef __GLIBC__    // e.g. OS with musl-libc
+typedef unsigned long __uint64_t;
+typedef signed long __int64_t;
```

Comment:
Thanks, I have replaced the usage of `__uint64_t` with `uint64_t`. Apparently, `__int64_t` isn't used anywhere.